scripts/verify_security.py

```
@@ -0,0 +1,101 @@
+    os.environ["FLASK_ENV"] = "development"
+    port = 5050
+    process = subprocess.Popen(
+        [sys.executable, "api_server.py"], env={**os.environ, "PORT": str(port)}
+    )
+
+    # Wait for server to start
+    time.sleep(3)
+
+    try:
+        url = f"http://localhost:{port}/health"
+        response = requests.get(url)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)